### PR TITLE
[FIX] Fix the code to support the python3 

### PR DESCRIPTION
**Pull request by JesusZapata**
Many compatibility to python3 over MQT

### CODE REVIEW

**Comment by JesusZapata** [comment]
@moylop260 
Your opinion is welcome

File: travis/git_run.py [hunk 1]
```diff
@@ -33,10 +34,10 @@ def get_items_changed(self, base_ref='HEAD'):
             e.g. "master" or "SHA_NUMBER"
         :return: List of name of items changed
         """
-        command = ['diff-index', '--name-only',
-                   '--cached', base_ref]
+        origin, branch = base_ref.split('/')
+        command = ['diff-index', '--name-only', '--cached', branch]
```
Comment (moylop260) [inline travis/git_run.py]: Why is required change `base_ref` by `branch` now? (And before not)

**Comment by JesusZapata** [comment]
@moylop260 
I revert the change not related with the `python3` support

**Comment by moylop260** [comment]
Could you create the MQT PR to supports py3?

**Pull request merged**